src/js/childwindow.js

```diff
@@ -129,6 +129,17 @@ var createChildWindow = function (event, url, frameName, disposition, options) {
     childview.webContents.loadURL(windowSettings.url);
   });
 
+  //support for copying current file URL to clipboard
+  electronLocalshortcut.register(childview, ["F8"], () => {
+    clipboard.writeText(windowSettings.url, "selection");
+    childview.webContents.loadURL(windowSettings.url);
+  });
+
+  electronLocalshortcut.register(childwin, ["F8"], () => {
```

Comment:
Should there only be one of these blocks?

Reply:
Also, can we add it to the menu? It looks like we could add it to `menu.js`, but I'm not sure how to access the window from there. If we put it in the menu, then Mac users will be able to override the keyboard shortcut.

Reply:
Ah, good points and thanks for feedback! I thought that adding it in `childwindow.js` was more logical since the file would open in a child window. Thus, you would only like the link of the child window. I'll take a look next weekend as I have a final coming up Thursday!